docs/docs/guides/customize-identity-model.mdx

```diff
@@ -0,0 +1,301 @@
+---
+id: customize-identity-model
+title: Customize Identity Model
```

Comment:
Sorry, I just saw this - while it is named "Identity Model" we really should change it to "Custom Identity Schema". We will change this in all the UIs etc. 

So please replace `Identity Model` with `Identity Schema`!

Reply:
I was going off this sentence in the identity concept docs:
`The Identity Model is a JSON Schema which describes your identity's (e.g. your customers / users / ...) fields.`

Do you think it would make sense to add the glossary from my other PR soon, for the different terms like Identity Model/Schema, Project Slug, etc: ?
https://github.com/ory/docs/blob/d0465fff31431b288ba9c1087a3757b7f160a6fc/docs/docs/concepts/self-service.mdx#glossary

I changed it, I hope it is clear in this document:
https://github.com/ory/docs/blob/a7438c42b2f0b837a9485d2a6d71c37ecc272953/docs/docs/concepts/identity.mdx